src/test/ui/command/command-create-pidfd.rs

```diff
@@ -0,0 +1,27 @@
+// run-pass
+// linux-only - pidfds are a linux-specific concept
```

Comment:
I think this is wrong. Shouldn't it be `only-linux`?

Reply:
Thanks for catching this!